### DISCUSSION TITLE
Set Environment Var to response body

### DISCUSSION THREAD

**Discussion by EdKing1375** [thunderclient/thunder-client-support#871] [Q&A]
Hi I am trying to set an environment var to a response body. The response body doesn't have any properties is it simply a string I was wondering how I can use Json query to just pull the response body's value. and example response is 
{
    "1234abcdefg"
}

**Comment by rangav** [comment]
Hi @EdKing1375 

Please following instructions in below link and image
https://github.com/rangav/thunder-client-support#setenv

<img width="1094" alt="Screenshot 2022-11-10 at 20 53 10" src="https://user-images.githubusercontent.com/8637550/201203688-06481277-312e-45c7-99aa-429dc8d50f4f.png">

**Comment by kshah-me** [comment]
Is there anyway to create variable on collection level ?
what I am looking for, Variable will be created once on collection level and value without change will be used into all test cases under collection ?
Note:- we don;t want to use any external files
